rel/overlay/etc/vm.args

```diff
@@ -108,9 +109,10 @@
 ## By default that would be "clouseau@127.0.0.1".
 ## Don't forget to override the paths to point to your certificate(s) and key(s)!
 ##
-#-proto_dist couch
-#-couch_dist no_tls '"clouseau@127.0.0.1"'
-#-ssl_dist_optfile <path/to/couch_ssl_dist.conf>
+-proto_dist couch
+-couch_dist no_tls '"clouseau@127.0.0.1"'
+-couch_dist no_tls '"node2@127.0.0.1"'
+-ssl_dist_optfile /Users/jiahuili/src/a/dev/couch_ssl_dist.conf
```

Comment:
These are for testing mostly, we don't want to enable this by default?

Reply:
yes, just for testing.